### PR TITLE
Remove powershell script from CI

### PR DESCRIPTION
**Pull request by bobbrow**
The current set of failures in the lab only repros when running the ci script.  This is an attempt to port the script to GitHub Actions.

### CODE REVIEW

File: .github/workflows/ci-main.win.yml [hunk 1]
```diff
@@ -11,18 +11,50 @@ jobs:
...
-      - name: Build the extension and run tests
-        run: pwsh -NonInteractive -NoProfile -NoLogo scripts/ci.ps1
+      - name: Build the extension
+        run: yarn compile-production
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Install CMake/Ninja
+        uses: lukka/get-cmake@v3.18.3
```
Comment (elahehrashedi) [inline .github/workflows/ci-main.win.yml:29]: did we choose this from GitHub Actions based on the number of stars?

File: .github/workflows/ci-main.win.yml [hunk 2]
```diff
@@ -11,18 +11,50 @@ jobs:
...
+
+      - name: Build fake compilers
+        uses: urkle/action-cmake-build@v1.0.0
+        with:
+          source-dir: ${{ github.workspace }}/test/fakeOutputGenerator
+          configure-options: -DCMAKE_INSTALL_PREFIX:STRING=${{ github.workspace }}/test/fakebin
```
Comment (elahehrashedi) [inline .github/workflows/ci-main.win.yml:35]: do we need this too?
` build-dir: ${{ github.workspace }}/test/build`

File: .github/workflows/ci-main.win.yml [hunk 3]
```diff
@@ -11,18 +11,50 @@ jobs:
...
+      - name: Build the extension
+        run: yarn compile-production
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Install CMake/Ninja
+        uses: lukka/get-cmake@v3.18.3
+
+      - name: Build fake compilers
```
Comment (elahehrashedi) [inline .github/workflows/ci-main.win.yml:31]: can we add a "test" somewhere in name, so we know it is a test?

File: .github/workflows/ci-main.win.yml (hunk 2 above)
Comment (elahehrashedi) [reply .github/workflows/ci-main.win.yml:35]: the test is passing though.

File: .github/workflows/ci-main.win.yml (hunk 1 above)
Comment (bobbrow) [reply .github/workflows/ci-main.win.yml:29]: yes

File: .github/workflows/ci-main.win.yml (hunk 3 above)
Comment (bobbrow) [reply .github/workflows/ci-main.win.yml:31]: ok

**Pull request closed without merging**